src/server/controllers/pokemon.js

```diff
@@ -0,0 +1,46 @@
+const { PrismaClient } = require('@prisma/client');
+const prisma = new PrismaClient();
+const axios = require('axios');
+
+const pokemonNumber = 151;
+const pokeArr = [];
+
+const createSinglePoke = (pokemon, arrIndex) => {
+    pokeArr.push({
+        name: pokemon.species.name,
+        art: pokemon.sprites.other.dream_world.front_default,
+        type1: pokemon.types[0].type.name
```

Comment:
What do you think about naming the "keys": "PrimaryType" and "SecondaryType"?

Reply:
I'll add to this: you could have a property called `types` which is just an array of types. This would tie in nicely to the comment I made about a `Type` model with relations